tests/testapps/CGCatalog/CGCatalog/CGPathAddCurveToPointViewController.m

```diff
@@ -36,13 +36,15 @@ - (void)loadView {
         CGContextSetStrokeColorWithColor(currentContext, self.options.lineColor);
         CGContextSetLineDash(currentContext, self.options.linePhase, self.options.lineDashPattern, self.options.lineDashCount);
 
+        CGAffineTransform transformation = CGAffineTransformIdentity;
+
         CGMutablePathRef thepath = CGPathCreateMutable();
-        CGPathMoveToPoint(thepath, NULL, 30, 100);
-        CGPathAddCurveToPoint(thepath, NULL, 47.0f, 67.0f, 50.0f, 55.0f, 45.0f, 50.0f);
-        CGPathAddCurveToPoint(thepath, NULL, 42.0f, 47.0f, 37.0f, 46.0f, 30.0f, 55.0f);
+        CGPathMoveToPoint(thepath, &transformation, 30, 100);
```

Comment:
Passing NULL here should work right?  We shouldn't have to change these files if we are not doing real transforms here.